src/main/kotlin/com/terraformation/backend/accelerator/db/Exceptions.kt

```diff
@@ -31,3 +33,7 @@ class SubmissionDocumentNotFoundException(id: SubmissionDocumentId) :
 
 class SubmissionNotFoundException(id: SubmissionId) :
     EntityNotFoundException("Submission $id not found")
+
+class ProjectVoteNotFoundException(projectId: ProjectId, phase: CohortPhase?, userId: UserId?) :
```

Comment:
Nit: This could move up a couple positions to keep the classes in alphabetical order (or at least to keep the "project" exceptions together).

Reply:
Also, maybe add `null` default values for the nullable arguments?